front/lib/api/assistant/actions/browse.ts

```diff
@@ -64,7 +64,7 @@ export class BrowseAction extends BaseAction {
     };
   }
 
-  renderForMultiActionsModel(): FunctionMessageTypeModel {
+  async renderForMultiActionsModel(): Promise<FunctionMessageTypeModel> {
```

Comment:
nit: should we remove the "MultiActions" part of the name ?

Reply:
This is orthogonal but yes we'll be able to clean that up